ui_framework/doc_site/src/services/routes/routes.js

```diff
@@ -164,6 +177,14 @@ const sandboxes = [{
   name: 'Kibana',
   component: KibanaSandbox,
   hasReact: true,
+}, {
+  name: 'FormGallerySandbox',
```

Comment:
How about changing this to "Inline forms"? I don't think we need the word "sandbox" in the title, and it seems like this view is dedicated to just inline forms, not all kinds of forms.